docs/software-design-federation-gateway-service.md

```diff
@@ -526,4 +548,40 @@ Proposal
 # Other Constraints and Conditions
 Timezone all times and dates are interpreted as timestamps in UTC (https://en.wikipedia.org/wiki/Coordinated_Universal_Time)
 
+# Appendix A
+
+Example self signed certificate that complies with above specification
+
+     openssl req -x509 -new \
+     		-days 365 \
+     		-newkey ec:<(openssl ecparam -name prime256v1) \
+     		-extensions v3_req \
+     		-subj "/CN=FGS Netherlands/OU=Corona-Team/O=Ministry of Public Health/emailAddress=operations@coronateam.nl/L=the Hauge/C=NL" \
+     		-outkey privkey.pem -nodes \
```

Comment:
`Unrecognized flag outkey`
Do you mean `-keyout` ?

Reply:
I thought I copied it from our scripts - wil double check.